network

```diff
@@ -103,19 +103,21 @@ function blockscout-help () {
 
 function devnet-help () {
     echo -e "${YELLOW}Devnet commands:${R}"
-    echo -e "  ${B}devnet pull${R}                      pull container images for all devnet services"
-    echo -e "  ${B}devnet start|up${R}                  start the network with default components"
-    echo -e "  ${B}devnet stop|down${R}                 shutdown the network and reset all database"
-    echo -e "  ${B}devnet allocations${R}               print information about pre-allocated wallets"
-    echo -e "  ${B}devnet state|status${R} --node       print lastest consensus state for all chains in the network"
-    echo -e "  ${B}devnet summary${R}                   print summary of the consensus state for all nodes in the network"
-    echo -e "  ${B}devnet restart${R}                   restart the chainweb-node service"
-    echo -e "  ${B}devnet ports${R}                     show ports of available services"
-    echo -e "  ${B}devnet height-details${R}            return detailed information for a given block height or 'latest' in JSON"
-    echo -e "  ${B}devnet services${R}                  show status of services in the network (docker compose ps)"
-    echo -e "  ${B}devnet chain-config${R}              show chainweb chain configuration"
-    echo -e "  ${B}devnet curl${R}                      run a curl command from within the docker network"
-    echo -e "  ${B}devnet reth-db${R}                   run a reth CLI database debugging command (first argument is the chain, i.e. 0 or 1)"
+    echo -e "  ${B}devnet pull${R}            pull container images for all devnet services"
+    echo -e "  ${B}devnet start|up${R}        start the network with default components"
+    echo -e "  ${B}devnet stop|down${R}       shutdown the network and reset all database"
+    echo -e "  ${B}devnet allocations${R}     print information about pre-allocated wallets"
+    echo -e "  ${B}devnet state|status${R}    print lastest consensus state for all chains in the network"
```

Comment:
Why is the `--node` option removed?

Reply:
It is not removed, but documented properly.

The previous documentation was misleading. There is no `--node` option. There is an optional parameter `[NODE]`. This is not the only command that takes (currently undocumented) options. Sprinkling those into the top level usage message, would just clutter it. Command specific options should be documented in the usage message for the respective command. (That is the same how other common programs with sub-commands do it, see, for instance, `git`, `docker`, or `cabal`).